front/lib/actions/mcp_actions.ts

```diff
@@ -96,12 +96,23 @@ function isEmptyInputSchema(schema: JSONSchema7): boolean {
 
 const MAX_TOOL_NAME_LENGTH = 64;
 
-const MAX_TEXT_CONTENT_SIZE = 2 * 1024 * 1024;
-const MAX_IMAGE_CONTENT_SIZE = 2 * 1024 * 1024;
-const MAX_RESOURCE_CONTENT_SIZE = 10 * 1024 * 1024;
+// Size limits for MCP tool outputs
+export const MAX_TEXT_CONTENT_SIZE = 2 * 1024 * 1024; // 2MB
+export const MAX_IMAGE_CONTENT_SIZE = 2 * 1024 * 1024; // 2MB
+export const MAX_RESOURCE_CONTENT_SIZE = 10 * 1024 * 1024; // 10MB
+
+export const MAXED_OUTPUT_FILE_SNIPPET_LENGTH = 32_000;
```

Comment:
these are characters right ? I would maybe double that so we are at roughly 16k tokens